instrumentation/rocketmq-client-4.8/testing/src/main/groovy/io/opentelemetry/instrumentation/rocketmq/AbstractRocketMqClientTest.groovy

```diff
@@ -123,8 +135,10 @@ abstract class AbstractRocketMqClientTest extends InstrumentationSpecification {
   def "test rocketmq produce and consume"() {
     when:
     runWithSpan("parent") {
-      producer.send(msg)
+      SendResult sendResult = producer.send(msg)
+      assert sendResult.sendStatus == SendStatus.SEND_OK
     }
+    tracingMessageListener.waitForMessages()
```

Comment:
```suggestion
    // waiting longer than assertTraces below does on its own because of CI flakiness
    tracingMessageListener.waitForMessages()
```